app/scripts/app.js

```diff
@@ -67,6 +67,21 @@
         controller: 'RoomsController',
         controllerAs: 'newRoom'
       })
+      .when('/rooms/pending', {
+        templateUrl: 'views/events/pending.html',
+        controller: 'EventsController',
+        controllerAs: 'event',
+        resolve:{
+          "check":function($location){
```

Comment:
Don't do it like this

`reslove` is used to resolve dependency.

Move admin checking into a service